### DISCUSSION TITLE
Remote Play Whatever crashing on repeat

### DISCUSSION THREAD

**Discussion by Zera213** [m4dEngi/RemotePlayWhatever#147] [General]
Hello, I'm having a very strange problem. 2 days ago I was able to use RPW with a friend on Tekken 8 with no problem. Fast forward to yesterday and today, I open RPW and send an invite to my friend and as soon as I click yes on the "Send remote play invite to (friend name)" remote play just shuts down instantly so I'm not sure what's happening. I tried reinstalling the new build and it didn't work, tried installing the 2.5 build and it did the exact same thing. I tried restarting my pc, turning off my windows defender and putting RPW in my exceptions list and it's still not working. Anyone have a solution or having the same problem?

**Comment by vbombatend** [comment]
Same here was working fine playing some other games with a friend but just as of i believe yesterday or the day before it isn't working anymore just crashes after trying to send invite or guest code, no matter what.

**Comment by Mikaaaay** [comment]
In this case, I noticed if Steam has an update, RPW won't work anymore. We will have to wait for a new version of RPW to match steam's version.
